src/views/SettingsView.vue

```diff
@@ -61,35 +62,41 @@ export default {
   },
 
   data() {
-    const profileProperties = [
+    let profileProperties = [
       {
         property: 'createdAt',
         text: 'Created at',
         formatter: formatDate,
       },
-      {
-        property: 'gasAllowanceLastResetAt',
-        text: 'Last allowance reset',
-        formatter: formatDate,
-      },
-      {
-        property: 'gasAllowanceRemaining',
-        text: 'Remaining gas',
-      },
-      {
-        property: 'gasAllowance',
-        text: 'Prepaid gas per month',
-      },
-      {
-        property: 'email',
-        text: 'Email address',
-      },
+
       {
         property: 'address',
         text: 'Ethereum address',
       },
     ]
 
+    if (config.supportEmailAccounts) {
+      profileProperties = profileProperties.concat([
```

Comment:
Instead of making `profileProperties` a `let`, you could keep it a `const` by just `push`ing the new values instead of using `concat`.

```suggestion
if (config.supportEmailAccounts) {
  profileProperties.push(
    {
      property: 'gasAllowanceLastResetAt',
      text: 'Last allowance reset',
      formatter: formatDate,
    },
    {
      property: 'gasAllowanceRemaining',
      text: 'Remaining gas',
    },
    {
      property: 'gasAllowance',
      text: 'Prepaid gas per month',
    },
    {
      property: 'email',
      text: 'Email address',
    },
  )
}
```

(Note that we're pushing a series of object, and not an anonymous array.)